cmd/maya-apiserver/cstor-operator/controller.go

```diff
@@ -138,45 +138,46 @@ func (cb *ControllerBuilder) Build() (*Controller, error) {
 	return cb.Controller, nil
 }
 
-// addSpc is the add event handler for spc.
+// addSpc is the add event handler for cspc.
 func (c *Controller) addSpc(obj interface{}) {
-	spc, ok := obj.(*apis.StoragePoolClaim)
+	cspc, ok := obj.(*apisv1alpha1.CStorPoolCluster)
```

Comment:
method name will need a change.

Reply:
Ok. But I have personally followed it from below link and is also intuitive for me. 
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/deployment_controller.go#L166

Reply:
cc @mittachaitu

Reply:
spc ---> cspc . will do it. thanks